### PR TITLE
Create minor-release-update.md

### PR DESCRIPTION
**Pull request by RubenOPS**
Creating a minor release update manual.

### CODE REVIEW

File: docs/minor-release-update.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+## Check latest version
+
+For the latest 4.y.z release check https://github.com/molgenis/molgenis-service-armadillo/releases/latest. This will redirect to a v4.y.z page.
+
+# Updating Armadillo
+
+### 1. Stop docker containers
+
+First, stop all profiles using the Armadillo UI.
```
Comment (timcadman) [inline docs/minor-release-update.md:13]: Could you add a little more detail where/how?

File: docs/minor-release-update.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+# should return empty list (i.e. default, xenon, rock)
+docker container list
+
+# remove containers not needed
+docker container stop <id>
+```
+
+## 2. Download required files
+
+Make a note of the version number ie. `v4.1.3` as you need to download some files from the terminal using the updatescript.
```
Comment (timcadman) [inline docs/minor-release-update.md:29]: Current version number?

Comment (timcadman) [inline docs/minor-release-update.md:29]: Should updatescript be update script?

File: docs/minor-release-update.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+systemctl start armadillo
+systemctl status armadillo
+```
+
+## 5. Log on to the UI
+
+Go to your armadillo website. Is the version in the left top corner updated? This means the update was successful. We're
+almost finished. 
+
+## 6. start profiles
```
Comment (timcadman) [inline docs/minor-release-update.md:132]: 'Start profiles'

**Review by timcadman — CHANGES_REQUESTED** [review]

**Review by timcadman — APPROVED** [review]

**Review by erikzwart — APPROVED** [review]
Functional test of local Armadillo update from 4.7.1 to 4.8.1, update was successful.
image

**Comment by marikaris** [comment]
Can we merge this? @RubenOPS

**Comment by RubenOPS** [comment]
Yes i think we can @marikaris

**Pull request merged**